.github/workflows/test.yaml

```
@@ -163,10 +163,10 @@ jobs:
           # We run selected test for GTK reactor inside X virtual framebuffer.
           - python-version: '3.7'
             tox-env: 'alldeps-gtk-withcov-posix'
-            trial-target: 'twisted.internet.test'
             platform-deps: 'gtk_platform'
             tox-wrapper: 'xvfb-run -a'
             job-name: 'gtk-tests'
+            trial-args: '--reactor=gi'
```

We have somewere above something like this

```
TRIAL_ARGS: "${{ matrix.trial-args || '-j 4' }}"
```

```suggestion
            trial-args: '--reactor=gi -j 4'
```

Right now, the GTK tests are not executed with disttrial and they take 10 minutes to run.

For Windows, we don't support distrial yet.

---

Yeah I realize I should probably have done that, I was just fighting the impulse to start cleaning this up since we should not be specifying "trial args" as one monolithic lump where we repeatedly configure parallelism, reactor, and whatever else individually for every part of the matrix... but I'll just go with this for now